bigbluebutton-html5/private/locales/en.json

```diff
@@ -290,6 +290,7 @@
     "app.actionsBar.actionsDropdown.createBreakoutRoom": "Create breakout rooms",
     "app.actionsBar.actionsDropdown.createBreakoutRoomDesc": "create breakouts for split the current meeting ",
     "app.actionsBar.actionsDropdown.takePresenter": "Take presenter",
+    "app.actionsBar.actionsDropdown.makePresenter": "Make presenter",
```

Comment:
I'd expect this to be part of the `app.userList.menu.*` batch, not sure how much it matters